cmd/uplink/cmd/setup_test.go

```diff
@@ -115,3 +120,59 @@ func TestDefaultPortAppliedToSatelliteAddrWithPortColonButNoPort(t *testing.T) {
 	assert.Equal(t, "ahost:7777", setupCmd.Flags().Lookup("satellite-addr").Value.String(),
 		"satellite-addr should contain default port when no port specified")
 }
+
+func TestSaveEncriptionKey(t *testing.T) {
+	expKey := make([]byte, rand.Intn(20)+1)
+	_, err := rand.Read(expKey)
+	require.NoError(t, err)
+
+	t.Run("ok", func(t *testing.T) {
+		dir, err := ioutil.TempDir("", "storj-test-cmd-uplink")
+		require.NoError(t, err)
+		defer func() { _ = os.RemoveAll(dir) }()
+
+		fname := filepath.Join(dir, ".enc.key")
```

Comment:
which is the convention for short var names?, I'm struggling to understanding it because yes, this one could be `fileName` or `filename` but then why `f` shouldn't be `file`?

I see a difference for the `expKey` reported above, because it's used across all the subtest functions and that's a longer scope, but these 2 are in the same scope and they aren't far away from their definition and usage.

Reply:
sorry, most probably I just missed `f`. Most of one char vars should be more descriptive.

Yeah its simple and small function but cost of renaming this is almost zero and it will help other devs to read your code. For you it's obvious. Also code can be reorganized later and scope can be larger. Its about keeping the same standards everywhere.

Reply:
> Yeah its simple and small function but cost of renaming this is almost zero 

Not replying for not doing, because of the cost to rename, because as you said, it's miserable, I wanted to understand what it's expected, to be align.

> and it will help other devs to read your code

That's subjective, because according other group of people, short variable names are clear for not long scopes, because names could confuse more than forcing the reader to find out exactly what the var contains rather than inciting them to assume it.

> Also code can be reorganized later and scope can be larger

I'm not a big fun on doing things for the sake of being a future unknown/unplanned refactoring.


> its about keeping the same standards everywhere

This the good point for doing it, hence, I'm going to review all the changes to have longer names.

---

A part of that, I would like to know the restrictions of having explicit names.

I suffered to have to long names for exported types, functions, etc., and in my opinion they were more a problem than a help, because the lines started to be too long and reading certain parts of code where there were too many of them was making harder to understand the code despite of the descriptive way.


For example when it's fine to use abbreviations and when not? I found places where `cfg` used as `configuration`, however in this review `expKey` has been renamed to `expectedKey` so which abbreviations are good and which ones aren't?

Reply:
> For example when it's fine to use abbreviations and when not? I found places where cfg used as configuration, however in this review expKey has been renamed to expectedKey so which abbreviations are good and which ones aren't?

I guess it is fine using abbreviations when they are common and unambiguous. For example, it's hard to misunderstand `cfg` - it is a widely used short name for "configuration". While `expKey` may not always be clear. Is it an "expected key", or an "expired key", or something else? `expKey` is not that common and may require some thinking what is it actually while reading it.

Reply:
you have to consider the context, it's a test, so you expect things, but I understand the point.

`cfg` I don't know if that's a common abbreviation, I used more `config` than `cfg`, but anyway, both are clear to me.